data-plane/scripts/amlight_sdx_lc_kytos.sh

```diff
@@ -4,15 +4,15 @@ set -eu
 sleep 30
 
 while ! nc -z 192.168.0.6 27027; do   
-  sleep 1 # wait 1 second before check for mongo1t again
+  sleep 5 # wait 1 second before check for mongo1t again
```

Comment:
Any reason why changing from 1s to 5 seconds? Since this does not seems to impose high overhead, having lower intervals would help detecting earlier when mongo is ready.

Reply:
I am allowing some time for MongoDB to initiate the replicas, a step that usually takes a few minutes. This preliminary phase is crucial to ensure the MongoDB instances are fully operational before starting Kytos.
In some cases, Kytos might encounter errors if the replicas are still being prepared. It is Leading to disruptions during Kyto's startup and impacting its functionality. To prevent such issues, we're taking this precautionary measure of ensuring that the MongoDB replicas are up and running seamlessly before initializing Kytos.